tt_metal/impl/kernels/data_types.hpp

```diff
@@ -24,8 +24,7 @@ enum NOC_MODE : uint8_t {
 };
 
 enum Eth : uint8_t {
-    SENDER = 0,
-    RECEIVER = 1,
+    ACTIVE = 0,
```

Comment:
while I agree w/ this rename, it is an API change which can break users' code.  we're close to doing a rename here w/ the API/corecoord changes, so think we should leave this be for now.